index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>AI for good</title>
+  <link rel="stylesheet" href="style.css"/>
+</head>
+
+<body>
+  <div class="home-container">
+    <header class="main-header">
+      <nav>
+        <div class="nav-bar">
+          <div class="top-nav-bar">
+            <a href="#"><img class="top-nav-item" src="assets/facebook.png" alt="facebook"></a>
+            <a href="#"><img class="top-nav-item" src="assets/twitter .png" alt="twitter"></a>
+            <a href="#" class="top-nav-item">English</a>
+            <a href="#" class="top-nav-item">My Page</a>
+            <a href="#" class="top-nav-item">Logout</a>
+          </div>
+          <div class="bottom-nav-bar">
+            <a href="index.html"><img class="nav-bar-logo" src="assets/ailogo.png" alt="AI-logo"></a>
+            <div class="nav-bar-links">
+              <a class="bottom-nav-link" href="about.html">About</a>
+              <a class="bottom-nav-link" href="#partner">Partner</a>
+              <a class="bottom-nav-link" href="#">Join</a>
+              <a class="bottom-nav-link" href="#">Sponsor</a>
+              <a class="bottom-nav-link" id="ai" href="#">BUY TICKETS</a>
+            </div>
+          </div>
```

Comment:
Your navigation bar on the `home` page is blurred as a result of the kind of background image you used for the header😉. and this doesn't match the design. Kindly ensure your nav bar is built to match the given design👍.

yours
![image](https://github.com/the-faizmohammad/conferencePage/assets/64643995/6e67089d-24c8-4c39-9609-3f544fab466c)

design
<img width="583" alt="image" src="https://github.com/the-faizmohammad/conferencePage/assets/64643995/7cecd798-1791-4f43-a87d-b84e59c4c55f">